src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/msquic_generated.cs

```diff
@@ -0,0 +1,2673 @@
+//
+// Copyright (c) Microsoft Corporation.
```

Comment:
So you're just going to manually copy this in every time you sync to the latest MsQuic?

Reply:
I also have to make all the classes/structs/enums internal, so I cannot just take it as it is.
We could publish the interop lib and consume that, but that would mean setting up infra for it and is not so straightforward.

Do you have any other way to consume this in mind?